src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
+- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api).
+- Tags are case sensitive.
+- Tags are defined by each client individually. The **list of tags** can be found in the [LiveChat Application](https://my.livechatinc.com/settings/tags). 
```

Comment:
```suggestion
- Tags are defined by each customer individually. The **list of tags** can be found in the [LiveChat Application](https://my.livechatinc.com/settings/tags). 
```

Reply:
Or maybe we should write "user"? WDYT, @oliwiapolec ?

Reply:
From the mentioned I think we should lean towards "user" (or maybe license, or admin? as only admins can alter the tags), we very rarely use "client" in the copy and the "customer" would be rather connoting the end-user

Reply:
So I think, in that case, I would go with 'admit' if only they could alter the tags. WDYT @ale-kacperczyk?

Reply:
My only objection to "admin" is that nowhere else are we so specific with roles in method descriptions. But 'user' might confuse readers to think that tags cannot be globally set by a license, but need to be created by each user. So, let's go with 'admin' and see what a reviewer from the API team says :)

Reply:
I would combine point starting with "To tag a chat..." with this one and go with
"you can list available tags here [link]... to create a new tag check/see [link]"
so there would be no mention of the user (and in case a "user" has to be mentioned then it should be admin, as only admins have sso-scopes that allow creating tags)

also pls drop
>To tag a chat, the **tag** needs to be created first.

this is like saying you need to create a thread first, otherwise, what will you tag? kinda obvious 😉

Reply:
It is obvious when you know it, but there are people who will try to use this method and forget that they need to create a tag first. How about:

> To tag a chat, the tag needs to be [created](/management/configuration-api/v3.5#create-tag) first.

This way, we will kill two birds with one stone (reminding people that they need to create a tag first & redirect them to the new tag methods).

Reply:
that's why I wanted to combine listing and creating tags in a single point... as for "tag needs to be created first", it's not an API-obvious but a life-obvious kinda thing

Reply:
You would think that it's life-obvious that you first need to register a property and only then are you able to update it, yet some users still try to update a non-existent property and look for property creation options in the Agent Chat API. The same might be true for tags and linking the method that allows for creating would help tackle that.

Okay, but I think merging the info about available tags & how to create tags into a single point is a nice compromise. 

In v3.2-v3.4, I would add:

Read how to [list all available tags](/management/configuration-api/v2.0#list-all-tags) and [create a new tag](/management/configuration-api/v2.0/#add-a-tag).

In v3.5, I would add a similar note but link to v3.5 methods instead.

Reply:
I would go with the compromise as I am also not so sure if it would be life-obvious for everyone.

Reply:
>Read how to [list all available tags](https://github.com/management/configuration-api/v2.0#list-all-tags) and [create a new tag](https://github.com/management/configuration-api/v2.0/#add-a-tag).

@ale-kacperczyk this is exactly what I had in mind 😉 
and my remark about obviousness is not about putting the above info in but about the "the tag needs to be created first" sentence... like duh

Reply:
OK, I get what you mean 👍🏻